Makefile

```diff
@@ -386,7 +387,8 @@ CI_DOC := doctool
 
 # Build and test addons without building anything else
 test-ci-native: LOGLEVEL := info
-test-ci-native: | test/addons/.buildstamp test/addons-napi/.buildstamp
+test-ci-native: | test/addons/.buildstamp test/addons-napi/.buildstamp \
+	benchmark/misc/function_call/build/Release/binding.node
```

Comment:
Probably needs to be added to `test-ci-js` since the benchmark is tested from JS. I'd suggest adding a comment since this dependency is not intuitive.